interview-questions.md

```diff
@@ -7,53 +7,71 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Are (most) Ruby methods accessors or mutators?
 
-  Your answer:
+  Your answer: In Learn bootcamp when we have Ruby methods class there we discussed about the ruby methods most of ruby methods are accessors 
 
-  Researched answer:
+  Researched answer: When I researched I learned that ruby methods are all accessors 
+  means they do not mutate objects they called on and for method to permanently change the object called on using bang operator
+ exp: num =[1,2,3,5]
+ num.reverse => [5,3,2,1]
+ num => [1,2,3,5]
+ num.reverse! => [5,3,2,1] used bang operator'!'
+ num => [5,3,2,1] permanantly changed
 
 
 
 2. What is a block in Ruby?
 
-  Your answer:
+  Your answer: I learned the word blocks in ruby blocks and loops method and wrote many programs with do and end commands in loops like javascript curly braces{}
 
-  Researched answer:
+  Researched answer: when I researched about blocks I learned that blocks are anonymous functions just like javascript and blocks are code that is passed to a method.Blocks can be created with do/end keywords 
```

Comment:
I think both of these answers together work very well

Reply:
thank you